RobotBase.java

```diff
@@ -54,6 +68,21 @@ public RobotContainerBase getRobotContainer() {
    */
   @Override
   public void robotInit() {
+    Logger.getInstance().recordMetadata("ProjectName", "2023-Robot"); // Set a metadata value
+
+    if (isReal()) {
+        Logger.getInstance().addDataReceiver(new WPILOGWriter("/home/lvuser")); // Log to a USB stick
```

Comment:
do we not have a usb drive?

Reply:
not really, i just found a 120mb flash drive in the kit of parts. We probably want a drive that is  a bit bigger like 4 or 8 gb so that we don't overwrite logs accidentally

Honestly even 1 gb should be fine